app/src/components/HorizontalPaginator.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Dimensions, ScrollView, View } from 'react-native';
+
+export interface HorizontalPaginatorProps {
+  items: JSX.Element[];
+}
+
+const HorizontalPaginator = (props: HorizontalPaginatorProps) => {
+  const screenDimensions = Dimensions.get('screen');
+
+  return (
+    <ScrollView
+      horizontal={true}
```

Comment:
I still think this view is not tall enough, but it's working right now. It's missing the following settings to make it better:
- Hide the horizontal scrollbar: 
```showsHorizontalScrollIndicator={false}```
- Add the transparent color for EdgeEffect in Android, on `android/app/src/res/values/styles.xml`:  
```
<item name="android:colorEdgeEffect">@android:color/transparent</item>
```